docs/docs/installation.md

```diff
@@ -0,0 +1,105 @@
+### Installing MuSpinSim
+
+In this tutorial we'll go through the best ways to install MuSpinSim on
+any system of your interest. The first step is, of course, procuring Python
+itself. If you have not already done this, see the [tutorial here](https://muon-spectroscopy-computational-project.github.io/tutorial-folder/python-setup).
+
+#### Dependencies
+
+`pip` will take care of installing all of the dependencies (other packages
+that `muspinsim` needs to run). Just follow the instructions below.
+
+#### Installing with pip or conda
+
+Assuming you already have Python and `pip` installed on your system, and a
+command line ready to use, then installing `muspinsim` is easy using
+
+```bash
+pip install muspinsim --user
+```
+or if you are using Anaconda, you may also use
+
+```bash
+conda install muspinsim
+```
+
+and you'll be ready to go.
+
+You do not need to worry about the details on the reset of this page unless the
+above failed, or you wish to install from the source.
+
+#### Installing from source
+
+To install from source you first need to ensure you have a suitable C++ compiler on
+your system. If you are on Linux or macOS, this is likely already the case, but for
+Windows you will likely need to install one such as [MSCV](https://visualstudio.microsoft.com/visual-cpp-build-tools/). 
+
+After this you should obtain the source from GitHub in one of two ways. If you have
+`git` installed and are familiar with its use you can simply clone the repository:
+
+```bash
+git clone https://github.com/muon-spectroscopy-computational-project/muspinsim.git
+```
+
+Otherwise, you can download it as a 
+[zip file](https://github.com/muon-spectroscopy-computational-project/muspinsim/archive/main.zip)
+and unzip it in a folder of your choice.
+
+To install the downloaded source, navigate to the parent directory in your
+terminal. If you don't know how to do it,
+[here's a handy guide for Linux and MacOS](http://linuxcommand.org/lc3_lts0020.php)
+and [here's one for Windows](http://dosprompt.info/basics.asp), which has a 
+different syntax. 
+
+Once you are in the directory *above* the unzipped/cloned package, you can run:
+
+```bash
+pip install ./{package} --user
+```
+
+where `{package}` represents the name of the directory where you unzipped/cloned
+the package. In your case, if you didn't rename the unzipped directory, it should be:
+
+```bash
+pip install ./muspinsim --user
+```
+
+##### Compiling with OpenMP
+
+To compile with OpenMP to allow parallelisation you can assign the environment variable
```

Comment:
```suggestion
If installing using `pip` or `conda`, then parallesisation with OpenMP will be enabled. If 
compiling from source, to allow parallelisation you can assign the environment variable
```

Think it might benefit from being clear about whether parallelism is enabled in the pip/conda versions, since we don't explicitly state it anywhere (also I'm assuming based on the other PR it is/will be, but correct this if it won't be).